bigquery/indexer.py

```diff
@@ -122,13 +130,14 @@ def index_facet_field(es, index_name, primary_key, project_id, dataset_id,
     table_name: BigQuery table name.
     field_name: BigQuery field name.
     readable_field_name: Field name for index and Data Explorer UI
+    billing_project_id: GCP Project ID to bill
```

Comment:
GCP project ID

"project ID" is what Cloud docs use.

Reply:
Done